lstchain/image/muon/muon_analysis.py

```diff
@@ -114,13 +156,17 @@ def fit_muon(x, y, image, geom, tailcuts):
     return ring, clean_mask, dist, image_clean
 
 
-def analyze_muon_event(subarray, event_id, image, geom, equivalent_focal_length,
-                       mirror_area, plot_rings, plots_path):
+def analyze_muon_event(subarray, tel_id, event_id, image, geom, equivalent_focal_length,
```

Comment:
You don't need to give `geom`, `equivalent_focal_length` and `mirror_area` when giving subarray and tel_id.

So this could be simplified now.